bson/raw_bson.py

```diff
@@ -13,6 +13,46 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+.. code-block:: python
+
+  import bsonjs
```

Comment:
Let's remove bsonjs since most people are not familiar with it and it's not needed to work with RawBSONDocuments.

Reply:
sounds good, rm'd